config.yml

```diff
@@ -475,10 +475,25 @@ nodes:
     fields:
       - name: left
         type: node
+        comment: |
+          Represents left side of expression.
```

Comment:
We need a little more information here. We know it's the left side of the expression by virtue of the name and the type. We want to know more than just that. Specifically: what kind of nodes can be in this position?

This comment applies to `left`/`right` in all four cases in this PR

Reply:
Updated